smoketests/__init__.py

```diff
@@ -216,6 +220,36 @@ def import_identity(self, identity, token, *, default=False):
         if default:
             self.spacetime("identity", "set-default", identity)
 
+    def subscribe(self, *queries, n):
+        self._check_published()
+        assert isinstance(n, int)
+
+        env = os.environ.copy()
+        env["SPACETIME_CONFIG_FILE"] = str(self.config_path)
+        args = [SPACETIME_BIN, "subscribe", self.address, "-n", str(n), "--print-initial-update", "--", *queries]
```

Comment:
Note to self: perhaps add a default timeout here, just so a wrong expectation doesn't make a test hang forever.